schema/src/ProjectInfo.yaml

```diff
@@ -0,0 +1,227 @@
+id: https://framework.regen.network/schema/ProjectInfo
+name: ProjectInfo
+prefixes:
+  linkml: https://w3id.org/linkml/
+  rfs: https://framework.regen.network/schema/
+  schema: http://schema.org/
+  qudt: http://qudt.org/schema/qudt/
+  unit: http://qudt.org/vocab/unit/
+  xsd: http://www.w3.org/2001/XMLSchema#
+imports:
+  - linkml:types
+default_curi_maps:
+  - semweb_context
+default_prefix: rfs
+default_range: string
+
+classes:
+  ProjectInfo:
+    class_uri: rfs:ProjectInfo
+    slots:
+      - name
+      - location
+      - description
+      - projectActivities
+      - projectSize # optional
+      - projectDeveloper # optional
+      - environmentType # optional
+      - projectMonitor # optional
+      - projectVerifier # optional
+      - projectStartDate # optional
+      - projectEndDate # optional
+      - creditClassVersion # optional
+  ProjectRole:
+    slots:
+      - name
+      - description
+      - url
+      - image
+    attributes:
+      type:
+        range: ProjectRoleTypes
+        slot_uri: rfs:ProjectRoleType
+        required: true
+  QuantityValue:
+    class_uri: qudt:QuantityValue
+    description: A value that represents a quantity, with a numeric value and a unit.
+    attributes:
+      numericValue:
+        slot_uri: qudt:numericValue
+        range: float
+        description: The numeric value of the quantity.
+      unit:
+        slot_uri: qudt:unit
+        range: string
+        description: The unit of measurement for the quantity.
+  Organization:
+    class_uri: rfs:Organization
+    slots:
+      - name
+      - url
+      - showOnProjectPage
+      - image
+    attributes:
+      description:
+        range: string
+        description: A description of the organization.
+  Location:
```

Comment:
@wgardiner I think to keep it in it's simplest form we should just have the location render as a lat-lon and leave things like bbox and mapbox contexts out. My thoughts here are that, though people might want to upload more complex geospatial info for the location, like a shapefile, most users right now will just upload a lat-lon location for the project. Will it affect any of our renderings if we just do 

`      "type": "Feature",
      "geometry": {
        "type": "Point",
        "coordinates": [
          -72.5,
          -2.5
        ]
`

Reply:
None of these are required fields, but this is the current structure of our location field on projects in our database, so that's what I modeled. I could change this to essentially support `any` content in the location field. We've also talked about migrating to WKT over GeoJSON (we're already using WKT for data post).

Reply:
afaict LinkML [does not currently support allowing arbitrary additional properties](https://github.com/linkml/linkml/issues/1404) to a slot. The only solution I've seen is to set an entire slot as `Any` type, which would mean we couldn't validate presence of `geometry`, `type`, and `place_name` (used by the frontend), so for now I recommend we keep these fields as optional data fields. I've simplified them a bit.